sdk/python/kfp/client/client.py

```diff
@@ -585,16 +585,21 @@ def get_experiment(self,
                     experiment_name))
         return result.experiments[0]
 
-    def archive_experiment(self, experiment_id: str):
+    def archive_experiment(self, experiment_id: str) -> None:
         """Archives an experiment.
 
         Args:
             experiment_id: id of the experiment.
+        """
+        self._experiment_api.archive_experiment(id=experiment_id)
 
-        Raises:
```

Comment:
Do we still have raises part for those?

Reply:
I could not get this to raise an exception, so I removed from the docstring. Test case:

```python
from kfp.client import Client

client = Client()
# does not raise an exception
assert client.archive_experiment(experiment_id="dummy_id") is None
```

Reply:
Sounds good, thank you!!